README.md

```diff
@@ -36,98 +28,49 @@ As a Maker
 
-Notes on functionality:
-------
+# Build status
 
-* You don't have to be logged in to see the peeps.
-* Makers sign up to chitter with their email, password, name and a username (e.g. samm@makersacademy.com, password123, Sam Morgan, sjmog).
-* The username and email are unique.
-* Peeps (posts to chitter) have the name of the maker and their user handle.
-* Your README should indicate the technologies used, and give instructions on how to install and run the tests.
+[![Build Status](https://travis-ci.com/m-budryte/chitter-challenge.svg?branch=master)](https://travis-ci.com/m-budryte/chitter-challenge)
 
-Bonus:
------
+# Screenshots
 
-If you have time you can implement the following:
+/pending/
 
-* In order to start a conversation as a maker I want to reply to a peep from another maker.
+# Tech/framework used
 
-And/Or:
+Sinatra
+Ruby 2.5.0
+RSpec
+Capybara
+Travis CI
+SQL
 
-* Work on the CSS to make it look good.
+# Tests
 
-Good luck and let the chitter begin!
+RSpec
+Capybara
+Travis CI
 
-Code Review
------------
+# How to use?
 
-In code review we'll be hoping to see:
-
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc.
-
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Referring to this rubric in advance may make the challenge somewhat easier.  You should be the judge of how much challenge you want this weekend.
-
-Automated Tests:
------
-
-Opening a pull request against this repository will will trigger Travis CI to perform a build of your application and run your full suite of RSpec tests. If any of your tests rely on a connection with your database - and they should - this is likely to cause a problem. The build of your application created by has no connection to the local database you will have created on your machine, so when your tests try to interact with it they'll be unable to do so and will fail.
-
-If you want a green tick against your pull request you'll need to configure Travis' build process by adding the necessary steps for creating your database to the `.travis.yml` file.
-
-- [Travis Basics](https://docs.travis-ci.com/user/tutorial/)
-- [Travis - Setting up Databases](https://docs.travis-ci.com/user/database-setup/)
-
-Notes on test coverage
-----------------------
-
-Please ensure you have the following **AT THE TOP** of your spec_helper.rb in order to have test coverage stats generated
-on your pull request:
+```
+Fork this repo
 
-```ruby
-require 'simplecov'
-require 'simplecov-console'
+$ bundle install
 
-SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new([
-  SimpleCov::Formatter::Console,
-  # Want a nice code coverage website? Uncomment this next line!
-  # SimpleCov::Formatter::HTMLFormatter
-])
-SimpleCov.start
+$ ruby app.rb
 ```
 
```

Comment:
Nice readme so far, maybe some intstructions on creating the database as I had to figure out which databases to create